### PR TITLE
MMCA 2741 -  Welsh translation for custom financials

### PR DESCRIPTION
**Pull request by NandanaRaghu**

### CODE REVIEW

File: conf/services/customs-financials.cy.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  - description: Pan mae’r sesiwn ar fin dod i ben, ni chaiff y defnyddiwr yr opsiwn o ymestyn ei sesiwn, ac nid oes rhybudd bod y sesiwn ar fin rhedeg allan o amser. Nid yw hyn yn bodloni maen prawf llwyddiant 2.2.1 (Timing Adjustable) Canllawiau Hygyrchedd Cynnwys y We, fersiwn 2.1
+    date: 2021-08-31
+  - description: O ran y meysydd mewnbwn ar gyfer y cyfeiriad a’r cyfeiriad e-bost, mae’r priodoleddau awtogwblhau ar goll, ac efallai y bydd hyn yn golygu na fydd y defnyddiwr yn gallu defnyddio’r data a gadwyd gan y porwr er mwyn llenwi’r meysydd hyn yn hawdd. Nid yw hyn yn bodloni maen prawf llwyddiant 1.3.5 (Identify Input Purpose) Canllawiau Hygyrchedd Cynnwys y We, fersiwn 2.1
+    date: 2021-08-31
+  - description: Pan gaiff y dudalen ei dangos yn Gymraeg, mae rhai cysylltiadau’n ymddangos yn Saesneg, ac nid oes ganddynt y briodoledd iaith berthnasol i ddangos eu bod mewn iaith wahanol. Nid yw hyn yn bodloni maen prawf llwyddiant 3.1.2 (Language of Parts) Canllawiau Hygyrchedd Cynnwys y We, fersiwn 2.1
+    date: 2021-08-31
+serviceLastTestedDate: 2021-04-12
+statementVisibility: public
+statementCreatedDate: 2019-09-16
+statementLastUpdatedDate: 2021-04-13
```
Comment (JoPintoPaul) [inline conf/services/customs-financials.cy.yml:36]: This date should be updated to `2021-05-04`

File: conf/services/customs-financials.cy.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+serviceName: Bwrw golwg dros eich cyfrifon ariannol ar gyfer y tollau
+serviceHeaderName: Bwrw golwg dros eich cyfrifon ariannol ar gyfer y tollau
```
Comment (JoPintoPaul) [inline conf/services/customs-financials.cy.yml:2]: `serviceHeaderName` is no longer used and can be deleted.

File: conf/services/customs-financials.cy.yml (hunk 1 above)
Comment (NandanaRaghu) [reply conf/services/customs-financials.cy.yml]: fixed

File: conf/services/customs-financials.cy.yml (hunk 2 above)
Comment (NandanaRaghu) [reply conf/services/customs-financials.cy.yml]: removed serviceHeaderName

**Review by JoPintoPaul — APPROVED** [review]

**Pull request merged**